onnxruntime_customops/mytorch/__init__.py

```diff
@@ -0,0 +1,41 @@
+"""
```

Comment:
It's a good thinking, but I don't expect these code will be re-used by any converter. And if it will, we can do it later. Right now, putting all pieces together will be more convenient.